integration_tests/jc-agent/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Must be run in interactive mode to activate;
+# Recommended run method:
+# docker run -dit --env JC_CONNECT_KEY=<your-connect-key-here> <container-reference-here>
+FROM ubuntu:20.04
+
+LABEL maintainer="github.com/cascadianblue"
+
+ARG version
+RUN test -n "${version}"
+
+ENV version=${version}
+LABEL version=${version}
+LABEL about.license="SPDX:Apache-2.0"
+
+# Install dependencies
+RUN apt-get update -qq -y && apt-get install --no-install-recommends -qq -y \
+        apt-transport-https \
+        curl \
+        ca-certificates \
+    && update-ca-certificates \
+    && apt-get -y autoclean \
+    && apt-get -y autoremove \
+    && rm -rf /var/lib/apt/lists/*
+
+# Kludgily install apt-show-versions
```

Comment:
Can you add a comment why this is necessary? Is it a dependency of the kickstart bootstrap script or do you just want it available to run from when running the interactive container (I don't have a JC_CONNECT_KEY which seems to be necessary to even view the script so can't tell if it is a requirement of that).